src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,174 @@
+package Duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+public class Duke {
+
+    public static void main(String[] args) throws IOException {
```

Comment:
Should try to avoid long methods that that goes beyond 30 lines of code. Could abstract into many smaller methods